Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -799,22 +800,62 @@ def redlock_get_scan_results():
         })
 
 
+def expire_stored_ids(fetched_ids):
+    """
+    Expires stored ids after 2 hours.
+
+    Args:
+        fetched_ids: list of fetched ids.
+
+    Returns:
+        The list of fetched ids.
+
+    """
+    if len(fetched_ids) == 0:
+        return []
+    cleaned_cache = []
+
+    two_hours = timedelta(hours=2).total_seconds() * 1000
+    now = int((datetime.utcnow() - datetime.utcfromtimestamp(0)).total_seconds() * 1000)
```

Comment:
why not just datetime.timestamp()?
https://docs.python.org/3/library/datetime.html#datetime.datetime.timestamp